src/gp_dag.cpp

```diff
@@ -106,6 +109,87 @@ EigenVectorXd GPDAG::BuildUniformQ() const {
   return q;
 }
 
+Node::NodePtrVec GPDAG::GenerateAllTrees() const {
```

Comment:
I realize I wasn't consistent about this, but I'd prefer to substitute every instance of "tree" in this file and with "topology". That's more accurate and is what I've tried to do elsewhere.

Reply:
Done.